docs/usage/setup-azure-devops.md

```diff
@@ -3,74 +3,77 @@ title: Azure DevOps setup
 description: How to setup Renovate for Azure DevOps
 ---
 
-# Azure DevOps setup (for npm/Yarn)
+# Azure DevOps setup
 
 1. Create a brand new pipeline within Azure DevOps, and select your source
    ![Azure DevOps create new pipeline](assets/images/azure-devops-setup-1.png)
 1. Select your repository
 1. Within _Configure your pipeline_ select: **Starter pipeline file**
    ![Azure DevOps starter pipeline template](assets/images/azure-devops-setup-2.png)
-1. Replace all contents with next npm/Yarn template (change all _PLACEHOLDER_ strings with your values):
+1. Replace all contents with:
 
    ```
    schedules:
      - cron: '0 3 * * *'
        displayName: 'Every day at 3am'
        branches:
-         include:
-           - master
-       always: true
+         include: [master]
 
    trigger: none
 
-   jobs:
-     - job:
-       pool:
-         vmImage: 'ubuntu-latest'
-       steps:
-         - checkout: none
+   pool:
+     vmImage: ubuntu-latest
 
-         - task: Bash@3
-           displayName: Create .npmrc file
-           inputs:
-             targetType: inline
-             script: |
-               cat > .npmrc << EOF
-               PLACEHOLDER(.NPMRC CONTENTS)
-               EOF
+   steps:
+     - task: npmAuthenticate@0
+       inputs:
+         workingFile: .npmrc
+   
+     - bash: |
+         git config --global user.email 'bot@renovateapp.com'
+         git config --global user.name 'Renovate Bot'
+         cp .npmrc ~/.npmrc
+         npx renovate
+       env:
+         TOKEN: $(System.AccessToken)
+   ```
+
+1. Add a file named `.npmrc` to your repository with the following contents:  
+   (replacing `YOUR-ORG` with your Azure DevOps organization and `YOUR-FEED` with your Azure Artifacts feed)
 
-         - task: npmAuthenticate@0
-           displayName: npm Authenticate
-           inputs:
-             workingFile: .npmrc
+   ```
+   @enpowerx:registry=https://pkgs.dev.azure.com/YOUR-ORG/_packaging/YOUR-FEED/npm/registry/ 
```

Comment:
Can you clarify here about what the registry should be? This example is for packages scoped under `@enpowerx`, but it might also be useful to have an example where the Azure Feed is used as a proxy for npm in general.

Reply:
I put the scoped sample here, because that's how I used it in my setup, but you're right the Azure Artifacts as a proxy for npm is probably the more common use-case. I've adjusted the PR accordingly.